packages/playwright-core/src/server/registry/browserFetcher.ts

```diff
@@ -36,7 +36,7 @@ export async function downloadBrowserWithProgressBar(title: string, browserDirec
 
   const zipPath = path.join(os.tmpdir(), downloadFileName);
   try {
-    const retryCount = 3;
+    const retryCount = 5;
```

Comment:
Here we either need to decide between:

a) Continuing trying only 3 times, so the other 2 mirrors are basically unused
b) Retrying up to 5 times so we keep using the other 2 mirrors

did b) for now.

Reply:
2) would be staying on the safe side and I vote for it.